android/BOINC/app/src/main/java/edu/berkeley/boinc/rpc/RpcClient.java

```diff
@@ -85,6 +88,8 @@
     public static final int MGR_SYNC = 31;
 
     private LocalSocket mSocket;
+    private Socket mTcpSocke;
```

Comment:
It's still a typo here:
`mTcpSocke` -> `mTcpSocket`

Reply:
Fix! :)